### ISSUE TITLE
Document the current use of `role` attribute in `particDesc`

### ISSUE THREAD

**Issue by cmil** [dracor-org/dracor-schema#81]
[RomDraCor](dracor-org/romdracor) (and in the future NeoLatDraCor) use the `role` attribute on `person` or `personGrp` elements in the `particDesc` to encode role descriptions. The rules for that application of the role attribute should be detailed in the ODD.

For some context see https://github.com/dracor-org/dracor-api/issues/275#issuecomment-2558053165 and https://github.com/dracor-org/neolatdracor/wiki/FAQs-on-Encoding-Plays-for-the-NeoLatDraCor-in-TEI#how-may-i-encode-stock-characters and

**Comment by cmil** [comment]
From what @juliajbeine describes in https://github.com/dracor-org/dracor-api/issues/275#issuecomment-2558053165 I would argue that a better way to do this would be to use the [roleDesc](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-roleDesc.html) element in [cast lists](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/DR.html#DRCAST). The cast list entries should then be linked to the particDesc entries via IDs so that the API would be able to extract role information (where available) for the characters in the particDesc. This approach would make clear where the data comes from and that it is not some external role classification scheme.

**Comment by lehkost** [comment]
In general, we can pursue this solution, linking castItem and person where it's possible (in fact, in GerDraCor, I'm slowly enhancing castList items to prepare for this step). In RomDraCor, though, the 20 comedies by Plautus that we feature, have no castLists (although castLists are present in our editions of Seneca's tragedies and Terence's comedies, 16 plays altogether). I'm not sure, @juliajbeine, that we can (or should) fix this by introducing artificial castLists where they are missing?

**Comment by cmil** [comment]
At least the printed edition of the Miles gloriosus has a list of PERSONAE, see https://archive.org/details/comoediaerecens00plaugoog/page/n13/mode/1up?view=theater (the link @juliajbaine provided in the [original issue](https://github.com/dracor-org/dracor-api/issues/275#issuecomment-2558053165) that sparked this one). And for cases where there is no cast list in the source edition the `role` attribute in `particDesc` entries may still be a viable option. The [TEI guidelines for that attribute](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-person.html#tei_att.role) also recommend defining the possible values using a `valList` element.

**Comment by juliajbeine** [comment]
Indeed, the TEI files derived from the PDL do not include all the information given in the text editions the files are based on. For instance, the lists of _dramatis personae_ and _argumenta_ are missing at least occasionally. We could consider adding this information to the TEI files in RomDraCor. What do you think @lehkost ?

I generally like the idea of using the roleDesc element in the castList element and linking it to the persons in the particDesc element. However, I noted that the roleDesc elements in the DraCor TEI files currently include punctuation marks, e.g., see 
https://dracor.org/api/v1/corpora/fre/plays/moliere-etourdi/tei
https://github.com/dracor-org/neolatdracor/blob/main/tei/macropedius-hecastus.xml
https://dracor.org/api/v1/corpora/ger/plays/anonym-das-urtheil-des-paris/tei 

For instance, the role descriptions ", servus", "servus.", and "servus" should all be processed as the same role of the "servus". Is there already a way to "filter" only the role description without any punctuation marks to link it to the persons in the particDesc element?

**Comment by cmil** [comment]
@juliajbeine The punctuation problem can be easily solved by fixing the markup. The castItems should be encoded like this:

```xml
<castItem>
  <role>Hecastus</role>,
  <roleDesc>maritus</roleDesc>.
</castItem>
```

**Comment by lehkost** [comment]
If we still draw the roles from the particDesc (with standardised values for the role attribute), then I would argue that the markup of the roleDesc information in the castList doesn't matter. Wording in this area can be very unreliable (abbreviations, plurals, different orthography), so I would not make our extraction of role types dependent on it nor on the markup used in this area, but build on the info in particDesc. This info is consolidated by the planned links between particDesc and castList.
